pkg/registry/autoscaler/rest.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
```

Comment:
An etcd registry is now a store.  See Derek's pull with namespaces, my changes to pods, or andys changes in OpenShift to image repositories.  You should look more like one of those (etcd is a separate package, rest is api object logic, etc).  See my email to the Kubernetes-dev google group